short-response.md

```diff
@@ -0,0 +1,95 @@
+# Problem Set 2.2 - Variables and Control Flow
+## Short Response Questions
+
+1. **What does the code below log? Why?**
+   ```javascript
+    {
+      let singer = 'Omar Apollo';
+    }
+
+    console.log(`My favorite singer is ${singer}`);
+   ```
+    The above code logs an error because the string variable `singer` is declared outside the scope of where the console can log it. 
```

Comment:
Good use of the word "scope".

More precise: "it was declared in an _inner scope_ and cannot be accessed in the global scope."